supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -152,12 +152,12 @@ The practice of having both modes together is often referred to as _hybrid_, _ag
 
 [discrete]
 [[bimonthly]]
-==== image:images/yes.png[yes] bimonthly (adverb)
-*Description*: _Bimonthly_ means every other month.
+==== image:images/no.png[no] bimonthly (adverb)
+*Description*: Do not use, because the word is ambiguous. _Bimonthly_ can mean either twice a month or every two months. Instead, write "twice a month" or "every two months" to remove ambiguity.
```

Comment:
I think the first sentence contains some redundancy. Do we need to state the term is ambigious twice? Recommend rewriting first sentence to "Do not use."

Reply:
Funny, that's how I had it and then added on the second part :). Will update to remove!